### PR TITLE
feat: add yearly memberships

### PR DESCRIPTION
**Pull request by jnizet**
Note that, to fully test it, you'll have to insert memberships for past years by hand in the database, to do as if globe was deployed for several years. Othewise, you won't see the past memberships.

### CODE REVIEW

**Comment by acrepet** [comment]
I've tested (adding row in the membership table), everything is perfect! Well done (the print form is nice :+1: )

**Comment by acrepet** [comment]
is it still WIP?

**Comment by jnizet** [comment]
@acrepet. All the tests have been added, and the branch is now rebased on master (which now includes the merged RUC PR). I'm just waiting for a review to merge this to master.

File: frontend/src/app/person-layout/person-layout.component.spec.ts [hunk 1]
```diff
@@ -37,25 +43,60 @@ describe('PersonLayoutComponent', () => {
...
+
+  it('should have a loading membership status initially', () => {
+    const fixture = TestBed.createComponent(PersonLayoutComponent);
+    fixture.detectChanges();
+
+    expect(fixture.componentInstance.membershipStatus).toBe('loading');
+    expect(fixture.nativeElement.querySelector('#membership-warning-icon')).toBeFalsy();
+  });
+
+  it('should change the working status when the current membership is loaded or when i is created or deleted', () => {
```
Comment (acrepet) [inline frontend/src/app/person-layout/person-layout.component.spec.ts]: when i is -> when it is

**Comment by acrepet** [comment]
My Review is done, I have just seen one typo :+1:

**Pull request merged**